src/components/shared/mobile-menu/mobile-menu.tsx

```diff
@@ -73,6 +73,8 @@ const MobileMenu = () => {
   const [openedDropdown, setOpenedDropdown] = useState(-1);
   const controls = useAnimation();
   const pathname = usePathname();
+  const hasLocalStorage =
+    typeof window !== 'undefined' && window.localStorage.getItem('isBanner') === 'true';
 
```

Comment:
To avoid console errors
```suggestion
  const [hasLocalStorage, setHasLocalStorage] = useState(false);

  useEffect(() => {
    setHasLocalStorage(typeof window !== 'undefined' && !!window.localStorage.getItem('isBanner'));
  }, [hasLocalStorage]);

```